expandybird/expansion/expansion.py

```diff
@@ -183,6 +194,99 @@ def _ValidateUniqueNames(template_resources, template_name='config'):
     # If this resource doesn't have a name, we will report that error later
 
 
+def IsTemplate(resource_type):
```

Comment:
Ville changed the definition of what this means, so you'll need to integrate with how he does it (i.e., it's a template if it exists as an import I think).